Casks/a/aria2d.rb

```diff
@@ -1,20 +1,15 @@
 cask "aria2d" do
-  version "1.3.8"
-  sha256 "f541b792ed813fa38ec72a87f42e8283383c781d0c9deac421bbc1cdababbb65"
+  version "1.3.9,433"
+  sha256 "efa4be7b0c0d47c814e8d8f8d392d760dd5a02ba46829136f8a6404dffc2bf6f"
 
-  url "https://raw.githubusercontent.com/xjbeta/AppUpdaterAppcasts/master/Aria2D/Aria2D%20#{version}.dmg",
-      verified: "githubusercontent.com/xjbeta/"
+  url "https://github.com/xjbeta/Aria2D/releases/download/#{version.csv.first}%28#{version.csv.second}%29/Aria2D.#{version.csv.first}.dmg"
   name "Aria2D"
   desc "Aria2 GUI"
   homepage "https://github.com/xjbeta/Aria2D"
 
-  # Older items in the Sparkle feed may have a newer pubDate, so it's necessary
-  # to work with all of the items in the feed (not just the newest one).
   livecheck do
     url "https://raw.githubusercontent.com/xjbeta/AppUpdaterAppcasts/master/Aria2D/Appcast.xml"
-    strategy :sparkle do |items|
-      items.map(&:short_version)
-    end
+    strategy :sparkle
   end
```

Comment:
@chenrui333 If it is possible that the pubDate may still be out of order in future we should retain the block.

```suggestion
  livecheck do
    url "https://raw.githubusercontent.com/xjbeta/AppUpdaterAppcasts/master/Aria2D/Appcast.xml"
    strategy :sparkle do |items|
      items.map(&:nice_version)
    end
  end
```